docs/README.md

```diff
@@ -3,6 +3,18 @@
 DataHub is LinkedIn's generalized metadata search & discovery tool. To learn more about DataHub, check out our 
 [LinkedIn blog post](https://engineering.linkedin.com/blog/2019/data-hub) and [Strata presentation](https://speakerdeck.com/shirshanka/the-evolution-of-metadata-linkedins-journey-strata-nyc-2019). You should also visit [DataHub Architecture](architecture/architecture.md) to get a better understanding of how DataHub is implemented and [DataHub Onboarding Guide](how/entity-onboarding.md) to understand how to extend DataHub for your own use case.
 
+In general, Datahub has two types of users in mind. One has metadata, and use tools provided by Datahub to ingest metadata into Datahub;  The other is to use Datahub to discover metadatas available within Datahub. Datahub provides intuitive UI, full text search capablitity, and graph relationship presentation to make the metadata discover and understanding much eaiser. 
+
+The following sequence diagram highlights the key features Datahub has, and how the two types of users - metadata ingestion engineers and metadata discover users, can take full advantage of the Datahub.
+
+![datahub-sequence-diagram](./imgs/datahub-sequence-diagram.png)
+1. It starts with ingesting your metadata into datahub. We provide a [collection of sample Python script](https://github.com/linkedin/datahub/tree/master/metadata-ingestion) for you. Those scripts work with the popular relationship databases, find metadata of the data source, and publish metadata in Avro data format to MetadataChangeEvent(MCE) Kafka topic.
+2. A MetadataChangeEvent(MCE) Processor consumes Kafka message with given topic, and make necessary transformation, send to Generalized Metadata Service(GMS), GMS persists the metadata to MySQL.
```

Comment:
Change MySQL to database since we now supports Postgres & MariaDB (and theoretically anything supported by Ebean)